src/dmd/aggregate.d

```diff
@@ -30,10 +29,8 @@ import dmd.globals;
 import dmd.id;
 import dmd.identifier;
 import dmd.mtype;
-import dmd.semantic2;
-import dmd.semantic3;
 import dmd.tokens;
-import dmd.typesem;
+import dmd.typesem : defaultInit;
```

Comment:
I know this is not the DMD style but that is the only function from there and I think it ought to be in mtype.d anyway, which I plan to follow up in another PR.

Reply:
Walter approved selective top-level imports in the past. I think he just doesn't see much value in them, but from what I gathered he isn't against them. So imho it's more than fine if it's just a few symbols - especially if it's one or two.

Reply:
Yeah I'm trying to figure out which ones use a single import and mark them so.

With this one specifically I think it should be in `mtype`.

Reply:
I don't think we should invest time in doing this manually except when writing code which adds new imports. IIRC people have already written tools for this. I think @CyberShadow has one?